ethereum/blockwatch/block_watcher.go

```diff
@@ -640,9 +640,6 @@ func (w *Watcher) filterLogsRecursively(from, to int, allLogs []ethtypes.Log) ([
 			if from == to {
 				return allLogs, fmt.Errorf("Unable to get the logs for block #%d, because it contains too many logs", from)
 			}
-
```

Comment:
The previous comment was pretty misleading, and should have been deleted in the linter PR. 

This wasn't dead code. The code that was previously here (deleted in a previous PR) was removed. This function does what is intended to do (return all of the logs in a range of blocks that it can), so there isn't a need for this comment anymore.